src/mixing.html

```diff
@@ -1,31 +1,112 @@
 <section data-max-toc="3">
- <h2 id="mixing">Mixing Markup Languages for Mathematical Expressions</h2>
+ <h2 id="mixing">Semantic Annotations</h2>
```

Comment:
Nice & brief, but annotations like svg or TeX aren't necessarily "semantic".

Reply:
I would second Bruce here.

"Foreign markup annotations" could be a neutral phrase. Just to offer one example.

Reply:
yes I think I'll change to `Annotating MathML`, don't need to say at this point point why or what we are annotating with